### ISSUE TITLE
[Bug] In public mode the Autoplay option of video widget does not work

### ISSUE THREAD

**Issue by prapullac** [appsmithorg/appsmith#5606]
Steps to reproduce:
1) Add a video widget
2) Navigate to the property pane
3) Toggle Auto play: True
4) Deploy the application
5) Navigate to "Share" 
6) Make App Public 
7) Copy the link 
8) open the application in a different browser
and observe the video player

Observation:
The video does not play automatically 

Expectation:
Video must play automatically as it is true on AutoPlay 

Video:
[![LOOM DEMO](http://cdn.loom.com/sessions/thumbnails/62df002c2c3640d786d2ec7d6b04a4f8-00001.gif)](https://www.loom.com/share/62df002c2c3640d786d2ec7d6b04a4f8)

**Comment by sbalaji1192** [comment]
Issue is observable on private apps as well.
Here's my finding,
This is happening because of the autoplay policies in place (https://developer.chrome.com/blog/autoplay/). 
TLDR;
1. Autoplay option works only if user has interacted with the current tab, hence why it works on the edit flow.
2. There is a work around to enable autoplay on page load, by muting the video by default.

**Comment by sbalaji1192** [comment]
@somangshu Although, I came across few inconsistent hacks to make it work, my suggestion is to let app developer know about this policy through a tooltip next to autoplay option in the property pane.

**Comment by somangshu** [comment]
I find the solution above acceptable, Push this to the backlog
